how-to/interact-storage.rst

```diff
@@ -230,45 +248,6 @@ Secret Access/Secret Key  secret from DSN value
 ------------------------  ---------------------------------------------------
 ========================  ====================  =============================
 
-s3cmd
```

Comment:
Do you suggest we not mention s3cmd at all?

Reply:
given awscli will work in all regions, it's not required.